CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ straightforward as possible.
 
 ### Changed
 
+- Updated README by replacing `loginNoRedirectNoPopup` with `openSignIn`
+- Updated tests by replacing `loginNoRedirectNoPopup` with `openSignIn`
+
```

Comment:
Added changes to the unreleased section since it makes no sense to publish a new version with docs changes. Merging to main will update our README correctly. This should be moved to the next release (probably the new loader).